simulated-sensor/simulated-sensor_shacl.ttl

```diff
@@ -0,0 +1,209 @@
+@prefix simulated-sensor: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/simulated-sensor/> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+simulated-sensor:SimulatedSensorShape a sh:NodeShape ;
+    sh:targetClass simulated-sensor:SimulatedSensor ;
+    # Ensure at least one sensor is available
+    sh:property [
+        skos:example "Radar" ;
+        sh:path simulated-sensor:sensorType ;
+        sh:minCount 1 ;
+        sh:maxCount 1;
+        sh:name "sensorType"@en ;
+        sh:or (
+            [ sh:node simulated-sensor:RadarShape ]
+            [ sh:node simulated-sensor:LidarShape ]
+            [ sh:node simulated-sensor:CameraShape ]
+        ) ;
+        sh:order 0 ;
+        sh:description "Perception and other relevant sensors for autonomous driving" ;
+    ] ;
+    # sensorIdentifier (string)
+     sh:property [
+        skos:example "RADAR" ;
+        sh:path simulated-sensor:sensorIdentifier ;
+        sh:datatype xsd:string ;
+        sh:minCount 0 ;
+        sh:maxCount 1 ;
+        sh:name "sensorIdentifier"@en ;
+        sh:order 1 ;
+        sh:description "Name/Identifier of the sensor in the simulator" ;
+    ] ;
+    # mountingPosition TODO: Perhaps this is too specific, not all simulators will use the same metric/parametrization for this
+    sh:property [
+        skos:example "Front-mid" ;
+        sh:path simulated-sensor:mountingPosition ;
+        sh:datatype xsd:string ;
+        sh:in ("Front-left" "Front-right" "Front-mid" "Top-left" "Top-right" "Top-mid" "Back-left" "Back-right" "Back-mid") ;
+        sh:minCount 0 ;
+        sh:maxCount 1 ;
+        sh:name "mountingPosition"@en ;
+        sh:order 2 ;
+        sh:description "Possible mounting positions of the radar" ;
+    ] . 
+
+# Available sensors
+# RADAR
+simulated-sensor:RadarShape a sh:NodeShape ;
+    sh:targetClass simulated-sensor:Radar ;
+    # numberOfBeams (integer)
+    sh:property [
+        skos:example "1500" ;
+        sh:path simulated-sensor:numberOfBeams ;
+        sh:datatype xsd:integer ;
+        sh:minCount 0 ;
+        sh:maxCount 1 ;
+        sh:minInclusive 0 ;
+        sh:name "numberOfBeams"@en ;
```

Comment:
`sh:name` ist sowas wie ein display Name, d. h. dieser sollte menschenlesbar angegeben werden. In diesem Fall `number of beams`. Das wird dann auch so beispielsweise im SD-Creation-Wizard angezeigt.
Diese Anmerkung bitte für alle `sh:name`s überprüfen.

Reply:
Fixed by 4dc7fbab1fd050acb8659276e8a87e60aede70dd